staff/templates/staff/reports/orga-report.tex

```diff
@@ -3,7 +3,10 @@
 \documentclass{article}
 \usepackage{orgacerts}
 
-% Pfad zum Ophasen-Wesen
+% Pfad zu Bildern
+\newcommand{\logoFS}{logo_fs.png}
```

Comment:
Due to vertical alignment issues I combined all images into single wide one, so this is therefore invalid.